models/pancreas_ct_dints_segmentation/configs/train.yaml

```diff
@@ -0,0 +1,353 @@
+---
+imports:
+- "$import glob"
+- "$import json"
+- "$import os"
+- "$import ignite"
+- "$from scipy import ndimage"
+input_channels: 1
+output_classes: 3
+arch_ckpt_path: "/workspace/MONAI/model-zoo/models/pancreas_ct_dints_segmentation/models/search_code_18590.pt"
```

Comment:
Here could you use the relative path, thus users do not need to override it and `bundle_root` at the same time? Thanks!
```
arch_ckpt_path: "$@bundle_root + '/models/search_code_18590.pt'"
```